libs/durability/near_full_data_storage.py

```diff
@@ -205,8 +214,8 @@ def delete_workload(workload_info_list: list, s3userinfo: dict, delete_percent:
         return resp
 
     # pylint: disable=too-many-arguments
-    @staticmethod
-    def perform_write_to_fill_system_percent(master_node: LogicalNode, write_per: int, s3userinfo,
+    def perform_write_to_fill_system_percent(self, master_node: LogicalNode, write_per: int,
+                                             s3userinfo,
                                              bucket_prefix, clients, bucket_list=None):
```

Comment:
please check if this can fit in above line

Reply:
Done